caniusepython3/overrides.json

```diff
@@ -6,6 +6,7 @@
     "django-social-auth": "https://pypi.python.org/pypi/python-social-auth",
     "dnspython": "https://pypi.python.org/pypi/dnspython3",
     "extras": "https://pypi.python.org/pypi/extras",
+    "flask-security": "https://github.com/mattupstate/flask-security/blob/master/.travis.yml",
```

Comment:
Can you change this to `http://pythonhosted.org/Flask-Security/changelog.html#version-1-7-3` ?